site/index.ts

```diff
@@ -133,7 +135,7 @@ domReady(() => {
 
     // asynchronously load the required bundles
     void import("../src/index").then(function ({ StacksEditor }) {
-        const editorInstance = new StacksEditor(place, content.value, {
+        const options: StacksEditorOptions = {
```

Comment:
Can we revert this change as well? That would allow us to remove the import at the top. Normally I wouldn't care, but I'm very cautious about adding additional imports to this file so I can keep the bundle separate to test lazy loading.

Reply:
But that would mean duplicating the options lines. Just checking if that's what you want? If it's ok, I'll revert the change.

Reply:
Ahhhh, ok that makes sense. It somehow went over my head that you were using it twice. This is perfectly fine then, sorry for the confusion!